lib/pf/UnifiedApi/Controller/Fingerbank/UserAgent.pm

```diff
@@ -0,0 +1,48 @@
+package pf::UnifiedApi::Controller::Fingerbank::UserAgent;
```

Comment:
Why duplicate controllers for files that contain exactly the same thing ?

Anything different here than from the other resources ?

Reply:
This is a mistake will fix